### PR TITLE
Add node-red examples to /contrib

### PR DESCRIPTION
**Pull request by sefgsefg**
**We made two examples that using Node-red to control kubeflow pipeline**

We use Node-red to create a user interface that allows users to select pipeline execution parameters through node settings. The process includes program code execution, YAML file compilation, automatic pipeline file upload, and data return, reducing user operation time.

Of course, users can also create their own Node-red interface through sample templates to meet their needs.

### CODE REVIEW

**Comment by juliusvonkohout** [comment]
Hello, you need to sign the DCO and we first need to get actual manifests or platform stuff, because this is the manifests/platforms repository. As explained in the manifests WG meetings similar to the Kserve integration.

https://github.com/kubeflow/manifests/tree/master/contrib/kserve there is UI integration and kustomize stuff with Kubernetes deployments etc.

**Comment by sefgsefg** [comment]
I have sign-off the DCO but it still get error. Is there any problem I don't solve?

**Comment by juliusvonkohout** [comment]
> I have sign-off the DCO but it still get error. Is there any problem I don't solve?

Please take a look at the output of the DCO check https://github.com/kubeflow/manifests/pull/2723/checks?check_run_id=25255505195

File: contrib/Node-red/Node-red/Horizontal-Federated-Learning/FL_kubeflow_with_node-red/example/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+ARG PREFIX
+
+ENV NODE_OPTIONS=--max_old_space_size=128
+
+USER root
+RUN apk update && \
+  apk add py3-pip python3-dev libffi-dev
+
+USER root
```
Comment (tzabbi) [inline contrib/Node-red/Node-red/Horizontal-Federated-Learning/FL_kubeflow_with_node-red/example/Dockerfile:11]: Why this line exsist? User root is already selected.

**Comment by juliusvonkohout** [comment]
I will soon close this due to inactivity.

**Comment by juliusvonkohout** [comment]
closed due to inactivity

**Pull request closed without merging**